src/Sandbox.php

```diff
@@ -134,6 +138,10 @@ public static function fromArray(array $in): self
         $sandbox->setExpirationAfterHours($in['expirationAfterHours'] ?? 0);
         $sandbox->setDeletedTimestamp($in['deletedTimestamp'] ?? '');
 
+        $sandbox->setDatabricksSparkVersion($in['databricks']['sparkVersion'] ?? '');
+        $sandbox->setDatabricksNodeType($in['databricks']['nodeType'] ?? '');
+        $sandbox->setDatabricksNumberOfNodes($in['databricks']['numberOfNodes'] ?? 0);
```

Comment:
just not sure if this can end in an undefined index error?

Reply:
No, the null coalescing operator takes care of it.

![CleanShot 2021-07-11 at 20 46 01](https://user-images.githubusercontent.com/215660/125206731-08256f00-e289-11eb-82f0-a6eb193d0b8c.jpg)